engine/access/rpc/backend/config.go

```diff
@@ -0,0 +1,59 @@
+package backend
+
+import (
+	"errors"
+	"time"
+
+	"github.com/onflow/flow-go/engine/access/rpc/connection"
+)
+
+// Config defines the configurable options for creating Backend
+type Config struct {
+	ExecutionClientTimeout    time.Duration                   // execution API GRPC client timeout
+	CollectionClientTimeout   time.Duration                   // collection API GRPC client timeout
+	ConnectionPoolSize        uint                            // size of the cache for storing collection and execution connections
+	MaxHeightRange            uint                            // max size of height range requests
+	PreferredExecutionNodeIDs []string                        // preferred list of upstream execution node IDs
+	FixedExecutionNodeIDs     []string                        // fixed list of execution node IDs to choose from if no node ID can be chosen from the PreferredExecutionNodeIDs
+	CircuitBreakerConfig      connection.CircuitBreakerConfig // the configuration for circuit breaker
+	ScriptExecutionMode       string                          // the mode in which scripts are executed
+}
+
+type ScriptExecutionMode int
+
+const (
+	ScriptExecutionModeLocalOnly ScriptExecutionMode = iota + 1
+	ScriptExecutionModeExecutionNodesOnly
+	ScriptExecutionModeFailover
+	ScriptExecutionModeCompare
```

Comment:
nit: include docs for `ScriptExecutionModeFailover` and `ScriptExecutionModeCompare` the other two are what it says on the label, of course.

Reply:
addressed in [d7bb319](https://github.com/onflow/flow-go/pull/4791/commits/d7bb319c7cb3ee58b24b28a74841b1db23b66a9c)